components/script/dom/document.rs

```diff
@@ -2571,7 +2571,7 @@ impl DocumentMethods for Document {
                 let USVString(url) = self.URL();
                 Ok(Root::upcast(StorageEvent::new_uninitialized(&self.window, DOMString::from(url))))
             },
-            "touchevent" =>
+            "touchevent" if PREFS.is_touch_events_enabled() =>
```

Comment:
Is it enough to handle it here? From [`compositor.rs`](https://github.com/servo/servo/blob/master/components/compositing/compositor.rs#L813) there seem to be another path that will eventually call `document.handle_touch_event()` without going though here?